fortran90-rules/src/main/resources/lex/COMPROJECTHeader.lex

```diff
@@ -104,90 +104,86 @@ STRING		 = \'[^\']*\' | \"[^\"]*\"
 	}
 	
 	private void raiseErrors() throws JFlexException {
-		if(this.linesType.isEmpty()){
```

Comment:
Raising an error in this specific rule and not in others seemed arbitrary, since checking empty files is not the purpose of this rule.
Having a blank file is possible (we found it in one of the projects we evaluated at EUMETSAT). Even though it doesn't present a great problem, such a file would be useless and should be avoided, the same way unused variables should be avoided. So, for this purpose we agree on having a specific rule that checks this situation. 
As an alternative, we would favor simply ignoring blank files, but not raise an error in an unrelated rule such as COMPROJECTHeader.